### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by othercodes** [quiqueporta/simple-value-object#6]
Hi there, im getting this error using this package:

```bash
/venv/lib/python3.6/site-packages/simple_value_object/value_object.py:120: DeprecationWarning: inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
  self._args, self._varargs, self._keywords, self._defaults = getargspec(method)
```

Python version: `3.6`
package version: `1.3`

**Comment by othercodes** [comment]
Looks like is solved in `master` branch. However latest  release `1.3.0` does not have the fix.

**Comment by othercodes** [comment]
Hi! it would be possible to create a package with the latest changes?

**Comment by quiqueporta** [comment]
Sorry, I have not seen the issue. I will try to fix it as soon as possible ;)
Thanks.

**Comment by quiqueporta** [comment]
@othercodes We have released a new version 1.4.0

**Comment by othercodes** [comment]
Awesome, thank you!
